pire/minimize.h

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
IMHO
I may be wrong, but this file looks like a copy-paste to me.
If it is indeed, I suggest refactoring it.